convert-image-format@el-amine-404/files/convert-image-format@el-amine-404/metadata.json

```
@@ -0,0 +1,7 @@
+{
+  "description": "Convert Image Format",
+  "uuid": "convert-image-format@el-amine-404",
+  "name": "Convert Image Format",
+  "author": "amine-el-houjjaji",
```

```suggestion
  "author": "el-amine-404",
```
This should be your GitHub username, matching the UUID.